### PR TITLE
Create XML skeleton

### PR DESCRIPTION
**Pull request by hectorerb**
This PR create XML skeleton

Ref: https://github.com/flyve-mdm/flyve-mdm-ios-inventory/issues/11

### CODE REVIEW

File: FlyveMDMInventory/DeviceInventory.swift [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+public class DeviceInventory: NSObject, XMLParserDelegate {
+    
+    func createXML() {
+        
+        let xmlInvetory = createDTD() +
+            createElement(
+                tag: "REQUEST",
+                value:
+                createElement(tag: "QUERY", value: "Inventory") +
+                    createElement(tag: "VERSIONCLIENT", value: "FlyveMDM-Agent_v1.0") +
```
Comment (ajsb85) [inline FlyveMDMInventory/DeviceInventory.swift]: The element: VERSIONCLIENT is an input.

**Comment by ajsb85** [comment]
@hectorerb I reviewed your PR

**Comment by hectorerb** [comment]
@hectorerb update!

**Comment by ajsb85** [comment]
@hectorerb Could you resolve the conflicts?

**Pull request merged**